src/main/kotlin/model/RacingCar.kt

```diff
@@ -5,9 +5,9 @@ const val MIN_NAME_LENGTH = 1
 const val MAX_NAME_LENGTH = 5
 
 class RacingCar(
-    private var moved: Int,
+    var moved: Int,
     private val generator: RandomNumberGenerator,
-    private val name: String
+    val name: String
```

Comment:
I changed the variable from private to public, if the variable have 'getter' method.
Because as I know, when the variable is public, Kotlin generate getter method automatically. 
So I though it's unnecessary to make private the variable and in this way, I can remove getter method in code.
How do you think about it? It's okay to make variable to public for generating getter method automatically?

Reply:
Yeah, it’s fine if all you need is to get data.
But I’m not really on board with using var when you're aiming for immutability.
So, if you need to keep your object immutable, how would you handle it?
Also, what are some advantages of using immutable objects?

Reply:
Unlike variables declared with val, variables declared with var can be modified externally. 
Therefore, making a var property public makes the object mutable. To ensure immutability, the var property should be declared as private. 
Allowing external modification of a class's variables can lead to unintended changes, even if those changes do not align with the intended responsibility of the object. Thus, making an object immutable helps maintain a state that is easier to debug and prevents arbitrary external modifications.